### DISCUSSION TITLE
Getting error when trying to `gd` (goto definition)

### DISCUSSION THREAD

**Discussion by jimrubenstein** [LazyVim/LazyVim#5196] [Q&A] [answered]
```
Error executing Lua callback: .../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:48: Vim:Error executing Lua callback: ...ovim/HEAD-b7782da/share/nvim/runtime/lua/vim/lsp/rpc.lua:291: Cannot serialise function: type not supported
stack traceback:
	[C]: in function 'encode'
	...ovim/HEAD-b7782da/share/nvim/runtime/lua/vim/lsp/rpc.lua:291: in function 'encode_and_send'
	...ovim/HEAD-b7782da/share/nvim/runtime/lua/vim/lsp/rpc.lua:338: in function 'request'
	...m/HEAD-b7782da/share/nvim/runtime/lua/vim/lsp/client.lua:677: in function 'request'
	...r/neovim/HEAD-b7782da/share/nvim/runtime/lua/vim/lsp.lua:868: in function 'buf_request'
	...r/neovim/HEAD-b7782da/share/nvim/runtime/lua/vim/lsp.lua:927: in function 'buf_request_all'
	...r/neovim/HEAD-b7782da/share/nvim/runtime/lua/vim/lsp.lua:948: in function 'buf_request_sync'
	...al/share/nvim/lazy/fzf-lua/lua/fzf-lua/providers/lsp.lua:496: in function 'fn_contents'
	...al/share/nvim/lazy/fzf-lua/lua/fzf-lua/providers/lsp.lua:662: in function <...al/share/nvim/lazy/fzf-lua/lua/fzf-lua/providers/lsp.lua:658>
	...s/jim/.local/share/nvim/lazy/fzf-lua/lua/fzf-lua/cmd.lua:35: in function 'run_command'
	...rs/jim/.local/share/nvim/lazy/fzf-lua/plugin/fzf-lua.lua:12: in function <...rs/jim/.local/share/nvim/lazy/fzf-lua/plugin/fzf-lua.lua:11>
	[C]: in function 'cmd'
	.../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:48: in function <.../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:16>
stack traceback:
	[C]: in function 'cmd'
	.../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:48: in function <.../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:16>
```

I'm getting this error when I try to `gd` on any node in a php code file, using intelephense.

Same happens in .vue files using volar (and tailwind, eslint, vtsls)

I've tried to update fzf, but that doesn't seem to help.

I'm not quite sure where to look to try and debug this.

Any insights?

**Comment by dpetka2001** [comment, marked as answer]
According to this `Error executing Lua callback: ...ovim/HEAD-b7782da/`, if I'm understanding correctly the error log you posted, it seems that you are using an old Neovim nightly version from May 21 2025. Try to upgrade to latest nightly version as in nightly there are always quite a lot of changes to APIs or use Neovim 0.10.2 stable instead. 

If you keep using nightly, I suggest you update pretty regularly. If you don't like upgrading Neovim regularly, then stick to a stable version.

**Reply by jimrubenstein** [reply]
Thanks for the reply!

I installed via brew. Add this to the list of reasons to get annoyed AF with brew.

I'll go reinstall another way and see how that works out.

Again, I really appreciate the response.

**Reply by dpetka2001** [reply]
You can use [bob](https://github.com/MordechaiHadad/bob) to install Neovim. Make sure you uninstall any previous Neovim you have on your system and follow the instructions on the Github repo to add it to your PATH. `bob` allows you to install both `nightly` and `stable` and you can also switch between them at your preference with just `bob use nightly` or `bob use stable`.

**Reply by jimrubenstein** [reply]
sir. you are a gentleman, and a scholar.

tyvm!
